tensorflow/python/framework/tensor_util_test.py

```diff
@@ -271,8 +271,8 @@ def testBfloat16(self):
   def testFloat8e5m2(self):
     test_type = dtypes.float8_e5m2.as_numpy_dtype
     t = tensor_util.make_tensor_proto(np.array([10.0, 20.0], dtype=test_type))
-    # 10.0: "I" = 73 = 10010 01: 2^(18 - 15) * (1 + 1/4)
-    # 20.0: "M" = 77 = 10011 01: 2^(19 - 15) * (1 + 1/4)
+    # 10.0: "I" = 73 = 0 10010 01: 2^(18 - 15) * (1 + 1/4)
```

Comment:
Nit: Why do we need the extra 0 here?

Reply:
Updated. (I just wanted to display all bit positions of FP8. But I agree that our repo should be consistent with the upstream repo as possible as we can, for less pain in the weekly sync.)